code/datums/outfits/jobs/engineering.dm

```diff
@@ -16,7 +16,6 @@
 	name = OUTFIT_JOB_NAME("Chief Engineer")
 	head = /obj/item/clothing/head/hardhat/white
 	uniform = /obj/item/clothing/under/rank/exultant
-	head = /obj/item/clothing/head/hardhat/white
```

Comment:
Why?

Reply:
No idea, wasn't even mentioned in the PR I pulled that from.